src/pages/home-page/home-page.css

```diff
@@ -44,10 +44,36 @@
   flex-flow: row wrap;
   gap:1rem;
   align-items: center;
+  justify-content: space-between;
+  width:100%;
+}
+.note-footer-end{
+  display:flex;
+  gap:0.5rem;
 }
 
 textarea{
   text-decoration: none;
   font-family: inherit;
   resize:vertical;
 }
+
+.addNoteCard{
+  top:8%;
+  grid-area: main;
+  position: relative;
+  width:100%;
+  height: 40rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(36, 15, 111);
+  border-radius: 0.5rem;
```

Comment:
Border radius should be used in pixels for consistent radius throughout all root font sizes.